src/System.Private.Reflection.Execution/src/Internal/Reflection/Execution/ExecutionEnvironmentImplementation.MappingTables.cs

```diff
@@ -439,6 +446,11 @@ public sealed override MethodInvoker TryGetMethodInvoker(RuntimeTypeHandle decla
                 // Now have the type loader build or locate a dictionary for this method
                 uint index = cookie >> 1;
 
+                if (!RuntimeAugments.SupportsRelativePointers)
+                {
+                    index = (uint)(IntPtr.Size * index) / sizeof(uint);
```

Comment:
Indexing into pBlob is becoming kind of awkward after this change. I think we should just change the type of pBlob to `void*` and cast to the right type before each use.

Reply:
Done. Thank you!